core/platform/email/mailgun_email_services.py

```diff
@@ -14,138 +14,108 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Provides mailgun api to send email."""
+"""Provides mailgun api to send emails."""
 
 from __future__ import absolute_import  # pylint: disable=import-only-modules
 from __future__ import unicode_literals  # pylint: disable=import-only-modules
 
 import base64
 
-from core.platform.email import gae_email_services
 import feconf
 import python_utils
 
 
-def post_to_mailgun(data):
+def send_email_to_recipients(
+        sender_email, recipient_emails, subject,
+        plaintext_body, html_body, bcc=None, reply_to=None,
+        recipient_variables=None):
     """Send POST HTTP request to mailgun api. This method is adopted from
     the requests library's post method.
 
     Args:
-        data: dict. The data to be sent in the request's body.
+        sender_email: str. The email address of the sender. This should be in
+            the form 'SENDER_NAME <SENDER_EMAIL_ADDRESS>' or
+            'SENDER_EMAIL_ADDRESS'. Must be utf-8.
+        recipient_emails: list(str). The email addresses of the recipients.
+            Must be utf-8.
+        subject: str. The subject line of the email, Must be utf-8.
+        plaintext_body: str. The plaintext body of the email. Must be utf-8
+        html_body: str. The HTML body of the email. Must fit in a datastore
+            entity. Must be utf-8.
+        bcc: list(str)|None. List of bcc emails.
+        reply_to: str|None. Reply address formatted like
+            “reply+<reply_id>@<incoming_email_domain_name>
+            reply_id is the unique id of the sender.
+        recipient_variables: dict|None. If batch sending requires
+            differentiating each email based on the recipient, we assign a
+            unique id to each recipient, including info relevant to that
+            recipient so that we can reference it when composing the email
+            like so:
+                recipient_variables =
+                    {"bob@example.com": {"first":"Bob", "id":1},
+                     "alice@example.com": {"first":"Alice", "id":2}}
+                subject = 'Hey, %recipient.first%’
+            More info about this format at:
+            https://documentation.mailgun.com/en/
+                latest/user_manual.html#batch-sending
+
+    Raises:
+        Exception: If the mailgun api key is not stored in
+            feconf.MAILGUN_API_KEY.
+        Exception: If the mailgun domain name is not stored in
+            feconf.MAILGUN_DOMAIN_NAME.
 
     Returns:
-        Response from the server. The object is a file-like object.
-        https://docs.python.org/2/library/urllib2.html
+        bool. Whether the email is "sent" successfully, contingent on the
```

Comment:
You don't need the inverted quotes around "sent" here. Mailgun does actually send the email.

What do you mean by the "contingent on the ..." part? If you're saying that True means "200", then you'll want to say "... successfully. (This is implied by the mailgun API returning a status code of 200.)" instead.